### PR TITLE
commit: add alias `ci` for cvs/svn/hg muscle memory havers

### PR DESCRIPTION
**Pull request by durin42**
As far as I can tell it won't hurt anyone to have this alias exist, but will help those of us with old muscle memory.

### CODE REVIEW

**Comment by PhilipMetzger** [comment]
Can you reword the commit message to `Commands: Alias commit to ci for cvs/mercurial/svn compatability`. ATM, the message is quite hard to read, eventhough I understand what you mean.

**Comment by durin42** [comment]
Done, though I have no idea if that'll get reflected right in the github UI. I'm an hg guy. :)

**Comment by PhilipMetzger** [comment]
Thanks, otherwise the patch looks great to me. I'd wait for martin's opinion though.

**Review by yuja — APPROVED** [review]
fwiw, I think commit message style is basically the same as Mercurial.

**Comment by martinvonz** [comment]
> fwiw, I think commit message style is basically the same as Mercurial.

Yes, I think @durin42's original message matched our (de facto) convention better. I think "muscle memory" or also more correct than "compatibility"; the latter makes me think of compatibility with scripts, which we don't care about. I can see how "muscle memory havers" could be hard to parse, however. "commit: add alias `ci` for users with muscle memory from cvs/svn/hg" might be easier to parse. Feel free to change or leave as is; I don't care that much about following any particular convention.

File: src/commands.rs [hunk 1]
```diff
@@ -366,7 +366,7 @@ struct DescribeArgs {
 
 /// Update the description and create a new change on top.
 #[derive(clap::Args, Clone, Debug)]
-#[command(hide = true)]
+#[command(hide = true, visible_aliases=&["ci"])]
```
Comment (martinvonz) [inline src/commands.rs:369]: I was wondering for a second what it means for the command to be hidden but the alias to be visible. I suppose it means that the alias will show up when you do `jj help commit`. Sounds good.

**Review by martinvonz — APPROVED** [review]

**Comment by martinvonz** [comment]
@durin42: Feel free to merge this PR
